mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/unmapped/C_smpzexmx net/minecraft/client/gui/DrawableHelper
 		COMMENT The texture of various icons and widgets used for rendering ingame indicators.
 	FIELD f_rlwncfgj STATS_ICON_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
 		COMMENT The texture of icons used in the stats screen.
+	FIELD f_wwyjxrzh LIGHT_DIRT_BACKGROUND_TEXTURE Lnet/minecraft/unmapped/C_ncpywfca;
```

Comment:
oh no; i was so confused by this name but then i realized: it's the actual name of the texture; i don't like it!